### PR TITLE
NAS-103465: Update rename tooltip to include allowable chars

### PR DESCRIPTION
**Pull request by q5sys**
Add information in the tooltip to inform users of valid characters.

### CODE REVIEW

File: src/app/helptext/system/bootenv.ts [hunk 1]
```diff
@@ -28,7 +28,11 @@ export const helptext_system_bootenv = {
   list_dialog_scrub_action: T("Start Scrub"),
 
   rename_name_placeholder: T("Name"),
-  rename_name_tooltip: T("Rename the existing boot environment."),
+  rename_name_tooltip: T(
+    "Rename the existing boot environment.\
+  Names may include alphanumeric characters as well as\
```
Comment (wblock) [inline src/app/helptext/system/bootenv.ts]: "May" implies permission. 
```suggestion
  Allowed characters are alphanumeric,\
```

**Comment by q5sys** [comment]
Updated as requested

**Comment by wblock** [comment]
See change as in #2894.

**Comment by q5sys** [comment]
Updated as requested.

**Pull request merged**